cmd/controller/app/start.go

```diff
@@ -67,14 +67,14 @@ TLS certificates from various issuing sources.
 It will ensure certificates are valid and up to date periodically, and attempt
 to renew certificates at an appropriate time before expiry.`,
 
-		// TODO: Refactor this function from this package
-		Run: func(cmd *cobra.Command, args []string) {
+		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := o.Validate(args); err != nil {
-				logf.Log.Error(err, "error validating options")
+				return fmt.Errorf("error validating options: %s", err)
```

Comment:
How does RunE handle displaying errors to users? Would we be better to use `Run` as before so that we can control the output format? Don't think it's a big deal, but I'm just uncertain how RunE actually works 😄

Reply:
RunE will output the error and then print the CLI flags. I think that makes sense here given failing on options.